packages/two_dimensional_scrollables/test/table_view/table_test.dart

```diff
@@ -0,0 +1,1135 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' show Platform;
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:two_dimensional_scrollables/table_view.dart';
+
+const TableSpan span = TableSpan(extent: FixedTableSpanExtent(100));
+const Widget cell = SizedBox.shrink();
+
+TableSpan getTappableSpan(int index, VoidCallback callback) {
+  return TableSpan(
+    extent: const FixedTableSpanExtent(100),
+    recognizerFactories: <Type, GestureRecognizerFactory>{
+      TapGestureRecognizer:
+          GestureRecognizerFactoryWithHandlers<TapGestureRecognizer>(
+        () => TapGestureRecognizer(),
+        (TapGestureRecognizer t) => t.onTap = () => callback(),
+      ),
+    },
+  );
+}
+
+TableSpan getMouseTrackingSpan(
+  int index, {
+  PointerEnterEventListener? onEnter,
+  PointerExitEventListener? onExit,
+}) {
+  return TableSpan(
+    extent: const FixedTableSpanExtent(100),
+    onEnter: onEnter,
+    onExit: onExit,
+    cursor: SystemMouseCursors.cell,
+  );
+}
+
+final bool masterChannel = !Platform.environment.containsKey('CHANNEL') ||
+    Platform.environment['CHANNEL'] == 'master';
+
+// TODO(Piinks): Remove once painting can be validated by mock_canvas in
+//  flutter_test
+// Contact Piinks if goldens need to be regenerated.
```

Comment:
Can we just document how to do it here? You know, in case of https://en.wikipedia.org/wiki/Bus_factor. 🚌  

(please stay safe!)

Reply:
LOL 🤞